### PR TITLE
Set closeMenuOnSelect to false. (Closes #1237)

### LINKED ISSUE #1237: Dont close the multi-select dropdown after an option has been selected
# Feature request
As a user, I get very frustrated needing to click the multiple option dropdown after I choose an option. I'd like if the dropdown stayed open until I hit the `esc` or `enter` key, tab away from the input, or until I click outside of the element.

## Proposed solution
Ensure that the [Select](https://github.com/OperationCode/front-end/blob/main/components/Form/Select/Select.js) component doesn't close its list of options after choosing one (if `isMulti` is true).

## Additional context
It's [react-select](https://github.com/JedWatson/react-select) under the hood.

## Demo of issue
demo

### PR DESCRIPTION
**Pull request by honsq90**
# Description of changes
Turns off closeMenuOnSelect for [Select.js](https://github.com/OperationCode/front-end/blob/main/components/Form/Select/Select.js)
https://react-select.com/props

# Issue Resolved

Fixes #1237

### CODE REVIEW

File: components/Form/Select/Select.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ export default function Select({
       <div className={styles.selectFeedbackGrouping}>
         <ThemedReactSelect
           {...props}
+          closeMenuOnSelect={false}
```
Comment (kylemh) [inline components/Form/Select/Select.js:118]: Looks like this is the right prop, but you'll wanna set it on the source code for ThemedReactSelect and only if `isMulti` is true.

Comment (honsq90) [reply components/Form/Select/Select.js:118]: ah yep, I'll do that

**Comment by kylemh** [comment]
The code looks perfect, but it doesn't appear to have worked. 🤔 

You can see for yourself via the input here: https://operation-code-h9q2lspps.vercel.app/code_schools

**Comment by honsq90** [comment]
> The code looks perfect, but it doesn't appear to have worked. 🤔
> 
> You can see for yourself via the input here: https://operation-code-h9q2lspps.vercel.app/code_schools

yea I'm a little stumped, I use react-select myself and that usually solves it. I'll have a look tomorrow, I wonder if it's re-rendering the whole component when a selection is made...

**Comment by honsq90** [comment]
hey @kylemh , I've tried this again via https://operation-code-h9q2lspps.vercel.app/code_schools and it's working as expected now. Wonder if something was cached?

**Comment by kylemh** [comment]
Just as in https://github.com/OperationCode/front-end/pull/1241 you have a correctly failing integration test! I'd need that to be fixed before merging this.

**Comment by kylemh** [comment]
We'll be disabling the rate limiting on the resources API tomorrow. Ignore the failing integration tests for `resources.spec.js`

I'll merge this tonight!

**Review by kylemh — APPROVED** [review]
Thanks so much for your 🎃 Hacktoberfest🎃&nbsp; contribution!

If you weren't aware, we're offering prizes too! You can read about the details [here](https://github.com/OperationCode/START_HERE#-hacktoberfest-).

Regardless, you've contributed, and we recommend you set yourself a reminder to submit the aforementioned form after November 1st (or after you meet all the criteria).

**Comment by honsq90** [comment]
Thanks @kylemh! The test now works locally btw

**Pull request merged**